server/request_intercepts.go

```diff
@@ -150,3 +152,48 @@ func (r *RpcRequest) intercept_eth_call_to_FlashRPC_Contract() (requestFinished
 	r.logger.Info("Intercepted eth_call to FlashRPC contract")
 	return true
 }
+
+func (r *RpcRequest) intercept_signed_eth_getTransactionCount() (requestFinished bool) {
```

Comment:
I would love if we could start doing unit / e2e tests for these handlers, but that seems def out of scope for now.